plugin/metrics/setup.go

```diff
@@ -98,3 +98,8 @@ const (
 	todo = 1
 	done = 2
 )
+
+// Based on Prometheus client_golang prometheus.DefBuckets
+var (
+	DefTimeBuckets = []float64{0.00025, 0.0005, 0.001, 0.0025 .005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
```

Comment:
This doesn't need be exported and can just be `var timeBucket` (also drop the `var ( )`)